packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.html

```diff
@@ -19,13 +19,17 @@
                         data-ng-bind="'network_security_dashboard_service_all' | translate"
                     ></option>
                 </select>
+                <span
+                    class="oui-icon oui-icon-chevron-down"
+                    aria-hidden="true"
+                ></span>
             </span>
             <span class="ml-2">
                 <oui-select
                     data-items="$ctrl.ipsList"
                     data-model="$ctrl.subnetSelected"
                     data-on-change="$ctrl.onSelectSubnet()"
-                    data-placeholder="{{:: 'network_security_dashboard_select_ip' | translate}}"
+                    data-placeholder="{{:: 'network_security_dashboard_select_select_ip' | translate}}"
```

Comment:
This is a suggestion : 

- you can replace **network_security_dashboard_select_select_ip** by **network_security_dashboard_select_ip** and the label is : Sélectionnez une adresse IP...

- you can replace **network_security_dashboard_select_ip** by **network_security_dashboard_select_search_ip** and the label is : Saisissez une adresse IP...

It could be more clear.